tony-core/src/test/resources/scripts/check_archive_file_localization.py

```diff
@@ -22,7 +22,9 @@
 if not os.path.isfile('./test.zip'):
     logging.error('test.zip doesn\'t exist')
     exit(-1)
```

Comment:
Will this fail now that you've changed `tony-core/src/test/resources/test.zip` to `tony-core/src/test/resources/test.zip::test20.zip` in the `testTonyResourcesFlag` test? Seems like there will no longer be a `test.zip`, only a `test20.zip`.

Reply:
yah.. that is the reason why tests are failling. Fixing